validator/src/main.rs

```diff
@@ -159,8 +161,28 @@ fn main() {
                 .long("snapshot-path")
                 .value_name("PATHS")
                 .takes_value(true)
+                .requires("snapshot_interval")
+                .requires("snapshot_package_output_path")
                 .help("Snapshot path"),
         )
+        .arg(
+            clap::Arg::with_name("snapshot_package_output_path")
+                .long("snapshot-package-output-path")
+                .value_name("OUTPUT_PATHS")
+                .takes_value(true)
+                .requires("snapshot_path")
+                .requires("snapshot_interval")
+                .help("Snapshot package output path"),
+        )
+        .arg(
+            clap::Arg::with_name("snapshot_interval")
```

Comment:
Lets make the units clear here please: snapshot_interval -> snapshot_interval_secs perhaps.